### ISSUE TITLE
Image Stylizing Distortions

### ISSUE THREAD

**Issue by justinej** [tensorflow/magenta#904]
Hi, I'm having troubles recreating some of the image stylizing results. I want to train their network to transform a single style ([Starry Night](http://hd.wallpaperswide.com/thumbs/the_starry_night-t2.jpg)) onto the ImageNet images. However, I am getting weird distortions that eventually obscure the content of the image. Does anyone have experience with this and have any tips about what to look at? Did anyone successfully train a model right out of the box?

If this is the wrong forum to be posting this kind of question, let me know and I can move it somewhere more appropriate. Also let me know if there's any other useful information you'd like me to include. Thank you!

I believe the problem is coming from the training process, because the stylization works fine when I use the pre-trained Monet checkpoint.  [Here](https://imgur.com/a/kd0ku) is what the transformations on a single image look like, over the progression of the training, with the number of steps in the captions. After 40,000 steps, we get:

![tiger_0_1000](https://user-images.githubusercontent.com/11280032/31062545-e4d51e98-a75d-11e7-824d-52799c7f6db2.png)

System desc: Ubuntu 16.04, CUDA V7.5.17, GeForce GTX 1060 6GB GPU

Only minor modifications to the code before running it:
* removed magenta imports. I'm straight up running the Python files, so that I can work with the local code (for debugging purposes)
* am using a special saver for the training step, to save the checkpoints that I want:
```
  train_saver = tf.train.Saver(
    max_to_keep=1,
    keep_checkpoint_every_n_hours=1.0/120)
```
The commands I am running:
```
python image_stylization_create_dataset.py
  --vgg_checkpoint=$VGG_CHECKPOINT
  --style_files=$STYLE_FOLDER
  --output_file=/tmp/image_stylization/style_images.tfrecord

python image_stylization_train.py
  --train_dir=/tmp/image_stylization/run1/train
  --style_dataset_file=/tmp/image_stylization/style_images.tfrecord
  --num_styles=1
  --train_steps=40000
  --batch_size=10
  --vgg_checkpoint=$VGG_CHECKPOINT
  --imagenet_data_dir=$IMAGENET

python image_stylization_transform.py
  --num_styles=1
  --checkpoint=$MODEL_CHECKPOINT
  --input_image=path/to/tiger/image
  --which_styles="{0:1.0}"
  --output_dir=/tmp/image_stylization/output
```
I've included the tensorboard loss graphs below. The loss values seem to be converging at abnormally high values (~1.0e5) which would suggest some kind of bug in the optimization code, maybe? 

![loss1](https://user-images.githubusercontent.com/11280032/31061977-4880a2a0-a759-11e7-9d19-f0a127736952.png)
![loss2](https://user-images.githubusercontent.com/11280032/31061978-48822954-a759-11e7-916e-1c99ea56c52b.png)

**Comment by wurstikon** [comment]
I had exactly the same problem when I tested training my own styles.

**Comment by xuguozhi** [comment]
mark
